client/extensions/woocommerce/state/wc-api/product-categories/test/selectors.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getProductCategories } from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#getProductCategories()', () => {
+		it( 'should return an empty array if data is not available.', () => {
+			const state = {
+				extensions: {
+					woocommerce: {}
+				}
+			};
+
+			expect( getProductCategories( state, 123 ) ).to.eql( [] );
+		} );
+
+		it( 'should give product categories from specified site', () => {
+			const categories123 = [
+				{ name: 'cat1' },
```

Comment:
Totally optional, but would you mind adding id and slug fields to these categories to make them look a little bit more like the actual response from WooCommerce?